src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_QuantumForceTransformer.java

```diff
@@ -0,0 +1,1160 @@
+    }
+
+    public GregtechMetaTileEntity_QuantumForceTransformer(final String aName) {
+        super(aName);
+    }
+
+    @Override
+    public IMetaTileEntity newMetaEntity(final IGregTechTileEntity aTileEntity) {
+        return new GregtechMetaTileEntity_QuantumForceTransformer(this.mName);
+    }
+
+    @Override
+    public String getMachineType() {
+        return "Quantum Force Transformer";
+    }
+
+    @Override
+    protected GT_Multiblock_Tooltip_Builder createTooltip() {
+        GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
+        tt.addMachineType(getMachineType())
+                .addInfo("Controller Block for the Quantum Force Transformer")
+                .addInfo("Allows Complex chemical lines to be performed instantly")
+                .addInfo("Requires 1 Catalyst Housing, all recipes need a catalyst")
+                .addInfo("All inputs go on the bottom, all outputs go on the top")
+                .addInfo("Accepts TecTech Energy and Laser Hatches")
+                .addInfo("Put a circuit in the controller to specify the focused output.")
+                .addInfo("Uses FocusTier*4*sqrt(parallels) Neptunium Plasma for focusing")
+                .addInfo("Can use FocusTier*4*sqrt(parallels) Fermium Plasma for additional chance output")
+                .addInfo("This multi gives bonuses when all casings of some types are upgraded")
+                .addInfo("Casing functions:")
+                .addInfo("Neutron Pulse Manipulators: Recipe Tier Allowed")
+                .addInfo("Neutron Shielding Cores: Focusing Tier")
+                .addPollutionAmount(getPollutionPerSecond(null))
+                .addSeparator()
+                .beginStructureBlock(15, 21, 15, true) // @Steelux TODO
+                .addController("Bottom Center")
+                .addCasingInfo("Bulk Production Frame", 96)
+                .addCasingInfo("Quantum Force Conductor", 177)
+                .addCasingInfo("Particle Containment Casing", 224)
+                .addCasingInfo("Neutron Shielding Cores", 234)
+                .addCasingInfo("Neutron Pulse Manipulators", 142)
+                .addInputBus("Bottom Layer", 4)
+                .addInputHatch("Bottom Layer", 4)
+                .addOutputHatch("Top Layer", 5)
+                .addOutputBus("Top Layer", 5)
+                .addEnergyHatch("Bottom Layer", 4)
+                .addMaintenanceHatch("Bottom Layer", 4)
+                .addStructureHint("Catalyst Housing", 4)
+                .addMufflerHatch("Top Layer (except edges), x21", 5)
+                .toolTipFinisher(CORE.GT_Tooltip_Builder);
+        return tt;
+    }
+
+    @Override
+    public IStructureDefinition<GregtechMetaTileEntity_QuantumForceTransformer> getStructureDefinition() {
+        if (this.STRUCTURE_DEFINITION == null) {
+            this.STRUCTURE_DEFINITION = StructureDefinition.<GregtechMetaTileEntity_QuantumForceTransformer>builder()
```

Comment:
Why is the structure definition defined here instead of a direct asignment to `STRUCTURE_DEFINITION `?

Reply:
Because there are methods, which are used, but aren't static. It's the same reason why, as an example, the MTE macerator's structure is done like that too. But not too sure on that one.

Reply:
Really just lazy initialization vs eager initialization. First one save a bit  of memory and a bit of startup time, later make it not a final field which is against my personal taste.